q2_taxa/plugin_setup.py

```diff
@@ -48,6 +61,10 @@
         'table': FeatureTable[Frequency]
     },
     parameters={'metadata': qiime2.plugin.Metadata},
+    input_descriptions={
+        'taxonomy': 'Taxonomic annotations for all features.',
```

Comment:
Can you incorporate the text I suggested above for the `taxonomy` input? Or some variation of it